Dockerfile

```diff
@@ -37,15 +30,14 @@ ENV JAVA_OPTS -Djava.security.egd=file:/dev/./urandom
 RUN ["/busybox/sh", "-c", "adduser -g '' -D zipkin"]
 
 # Add environment settings for supported storage types
+ENV KAFKA_STORAGE_VERSION 0.5.1-SNAPSHOT
```

Comment:
note: you can also use jitpack for this.. it would allow you to save others from a local build. another change of course. https://github.com/openzipkin/docker-zipkin/pull/212